src/gui/configurationTabFuncs.cpp

```diff
@@ -127,6 +130,10 @@ void ConfigurationTab::updateControls()
     ui_.AtomPopulationLabel->setText(QString::number(configuration_->nAtoms()));
     ui_.MoleculePopulationLabel->setText(QString::number(configuration_->nMolecules()));
 
+    // Applied size factor
+    ui_.SizeFactorLabel->setText(QString::number(configuration_->appliedSizeFactor().value_or(0.0)));
```

Comment:
Picky, I know, but the `0.0` doesn't represent a valid size factor (even if it isn't being shown!)
```suggestion
    ui_.SizeFactorLabel->setText(configuration_->appliedSizeFactor() ? QString::number(*configuration_->appliedSizeFactor() : "N/A"));
```

Reply:
I was getting ready to defend this, but I think your option is better.